src/main/java/gov/nih/ncats/molwitch/cdk/CdkChemicalImpl.java

```diff
@@ -2438,8 +2500,9 @@ public void remove() {
 			
 		};
 	}
-	
-	
-	
 
+	@Override
+	public Class getFactoryClass() {
```

Comment:
I'm 99.9% sure we won't need this. We can get the factory from the ImplUtil class if needed, and I don't think we need to go from a ChemicalImpl to its factory? I'd rather not add such a method unless we're sure we need it.